components/web/apps/org.wso2.carbon.apimgt.store/src/main/public/templates/appListRow.hbs

```diff
@@ -0,0 +1,35 @@
+<tr role="row" class="odd">
+    <td class="sorting_1">
+        <a title="applications"
+           href="/store/application/{{name}}/">{{name}}</a>
+    </td>
+    <td>{{tier}}</td>
+    <td>{{status}}</td>
+    <td>{{subscriptions}}</td>
+    <td>
+        <a title="View" href="/store/application/{{name}}"
+           class="btn  btn-sm padding-reduce-on-grid-view">
+      <span class="fw-stack">
+          <i class="fw fw-ring fw-stack-2x"></i>
+          <i class="fw fw-view fw-stack-1x"></i>
+      </span>
+            <span class="hidden-xs">View</span>
```

Comment:
With new UUF features, files within public folder will not be used. This code will be removed.